configs/server/forseti_conf_server.yaml.sample

```diff
@@ -1,41 +1,40 @@
 global:
 
-    # database
-    db_host: DB_HOST
-    db_user: DB_USER
-    db_name: DB_NAME
-
-    # gsuite
-    groups_service_account_key_file: GROUPS_SERVICE_ACCOUNT_KEY_FILE
-    domain_super_admin_email: DOMAIN_SUPER_ADMIN_EMAIL
-
-    # notification
     # Remove or comment out the email properties if you do not want Forseti to
     # send email notifications.
     email_recipient: EMAIL_RECIPIENT1,EMAIL_RECIPIENT2
     email_sender: EMAIL_SENDER
     sendgrid_api_key: SENDGRID_API_KEY
 
-    # quota
+##############################################################################
+
+inventory:
+
+    # Root resource to start crawling from, formatted as
+    # <resource_type>/<resource_id>, (e.g. "organizations/12345677890")
+    root_resource_id: ROOT_RESOURCE_ID
+
+    # gsuite
+    groups_service_account_key_file: GROUPS_SERVICE_ACCOUNT_KEY_FILE
```

Comment:
nit: can we change both of these to `gsuite_service_account_key_file`; group wouldn't be accurate as we are also using the same key to get users.

Reply:
Done.